frontend/src/components/MapView/LeftPanel/AnticipatoryActionPanel/utils.tsx

```diff
@@ -200,18 +150,85 @@ export function getAAIcon(category: AACategoryType, phase: AAPhaseType) {
   }
 }
 
+export function getAAIcon(category: AACategoryType, phase: AAPhaseType) {
+  const background = getAAColor(category, phase);
+  const otherProps = (() => {
+    switch (category) {
+      case 'na':
+        return { topText: 'na', bottomText: '-', color: 'black' };
+      case 'ny':
+        return { topText: 'ny', bottomText: '-', color: 'black' };
+      case 'Severo':
+        if (phase === 'Set') {
+          return {
+            topText: 'S',
+            bottomText: 'SEV',
+            color: 'white',
+          };
+        }
+        if (phase === 'Ready') {
+          return {
+            topText: 'R',
+            bottomText: 'SEV',
+            color: 'white',
+          };
+        }
+        throw new Error(`Icon not implemented: ${category}, ${phase}`);
+
+      case 'Moderado':
+        if (phase === 'Set') {
+          return {
+            topText: 'S',
+            bottomText: 'MOD',
+            color: 'black',
+          };
+        }
+        if (phase === 'Ready') {
+          return {
+            topText: 'R',
+            bottomText: 'MOD',
+            color: 'black',
+          };
+        }
+        throw new Error(`Icon not implemented: ${category}, ${phase}`);
+
+      case 'Leve':
+        if (phase === 'Set') {
+          return {
+            topText: 'S',
+            bottomText: 'MIL',
+            color: 'black',
+          };
+        }
+        if (phase === 'Ready') {
+          return {
+            topText: 'R',
+            bottomText: 'MIL',
+            color: 'black',
+          };
+        }
+        throw new Error(`Icon not implemented: ${category}, ${phase}`);
+
+      default:
+        throw new Error(`Icon not implemented: ${category}, ${phase}`);
+    }
+  })();
+
+  return <AAIcon background={background} {...otherProps} />;
+}
+
 export function AADataSeverityOrder(
```

Comment:
The severity order is actually a bit different I learned.
"Set" always takes precedent, so for example "Set Moderate" > "Ready Severe"